py/desitarget/io.py

```diff
@@ -1293,7 +1293,7 @@ def write_skies(targdir, data, indir=None, indir2=None, supp=False,
         else:
             subpseed = 718
         if hpxlist is not None:
-            subpseed += int(hpxlist[0])
+            subpseed += int(np.atleast_1d(hpxlist)[0])
```

Comment:
This looks sensible, but the docstring does specify that `hpxlist` should be a list, so the situation you're guarding against isn't meant to occur. Please update the docstring to specify that `hpxlist` _can_ be passed as an integer.

Reply:
Done in https://github.com/desihub/desitarget/pull/788/commits/83c8a2421dcb6f91550b717804e57c7985cfcf0d.